add-repos.sh

```diff
@@ -64,6 +64,7 @@ do
     BUILD_ACTION=""
     SKIP=""
     SKIP_REASON=""
+    CUSTOM_COMMAND=""
```

Comment:
I think `BUILD_ACTION` already provides this in the seed.groovy

buildAction | Optional | Additional build tool tasks/targets to execute

Reply:
I see your point, so for `OpenRefine`, we just need to add it to the 5th position

from
```
OpenRefine/OpenRefine,master,11,,maven,,,
```
to
```
OpenRefine/OpenRefine,master,11,,maven,-Dmaven.antrun.skip=true -pl !packaging/pom.xml,,
```

Reply:
Yep. I just updated the readme to reflect all the options that are supported.

Reply:
Thanks, I closed this one and pulled up https://github.com/moderneinc/jenkins-ingest/pull/42 instead.